index.html

```diff
@@ -30,6 +30,144 @@ <h1>Frehiywot <br />Nega</h1>
     </p>
     <a href="#start_collaboration" class="collaboration">Start collaboration</a>
     <br>
-    <div class="image_geometry_2"></div> 
+  
+    <h2>Projects</h2>
+  </section> 
+
+  <section class="Works-section" >
+   
+      <div class="worksection1"> </div>
+      <div class="rectangle">
+        <h3>Project name goes here</h3>
+        <br>
+        <ul class="buttons">
+         
+          <li><button class="btn HTMLCSS">HTML/CSS</button></li>
+          <li><button class="btn Ruby">Ruby on Rails</button></li>
+          <li><button class="btn Js ">JavaScript</button></li>
```

Comment:
- [ ] As the previous reviewer mentioned please make sure you stick to the Figma design. You should remove the bullets points from the list item so that the list items can be properly aligned 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/191843841-edf9bbb1-9bd6-436e-9bfd-7bf415150827.png)